react/states_and_effects/more_on_state.md

```diff
@@ -148,7 +152,7 @@ const handleIncreaseAge = () => {
 
 When a callback is passed to the `setState` function, it ensures that the latest state is passed in as an argument to the callback. 
 
-You will find yourself reaching out for state updater functions rarely, but it's good to know they exist.
+Using an updater is not always necessary. If you want to change the state using your previous state, and you prefer consistency over verbosity then you might consider using an updater. [is using an updater always preferred?](https://react.dev/reference/react/useState#is-using-an-updater-always-preferred)
```

Comment:
I don't think we need to link the react.dev resource here. Even if we do it should be in additional resources, what do you think?

Reply:
I wanted to put links to make the students more confident of the recommendation that we are providing, and let them know where our recommendations are coming from.

I have moved both this and the link above to additional resources, although I wasn't sure the name `React tip` was the appropriate tag, could be changes, or I could remove both deep dive links if you think they are unnecessary detail at this point.

Reply:
Left a comment on this below.